projects/1.20-fabric/assets/neepmeat/neepmeat/lang/zh_cn.json

```
@@ -819,6 +819,10 @@
   "item.neepmeat.spider_legs": "六足虫",
   "implant.neepmeat.spider_legs": "六足虫",
   "implant.neepmeat.spider_legs.desc_0": "增加速度，让完整方块上的移动更丝滑。",
+  "hexapod_skin.neepmeat.default": "默认",
+  "hexapod_skin.neepmeat.th_0": "跨性别袜",
+  "hexapod_skin.neepmeat.th_1": "自创编程袜",

```

```suggestion
  "hexapod_skin.neepmeat.th_1": "原创节目袜",
```
依据：https://en.wikipedia.org/wiki/Original_programming